horizon/responses.go

```diff
@@ -31,6 +31,16 @@ type Account struct {
 	Signers              []Signer          `json:"signers"`
 }
 
+func (a Account) GetNativeBalance() string {
```

Comment:
If the account doesn't exist you won't get `Account` response. `LoadAccount` will return error.